lang/lpython/Portfile

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               compiler_blacklist_versions 1.0
+PortGroup               github 1.0
+
+github.setup            lcompilers lpython 0.20.0 v
+revision                0
+categories              lang python
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 BSD
+description             High-performance typed Python compiler
+long_description        LPython aggressively optimizes type-annotated Python code. \
+                        It has several backends, including LLVM, C, C++ and WASM. \
+                        LPython’s primary tenet is speed.
+homepage                https://lpython.org
+
+checksums               rmd160  01cdab5767af01d017339f79355c34e9ab2ec8d7 \
+                        sha256  741fb64ddf70528eb713103d5b6e2068344b5768b546b0c21145d5a1507635a0 \
+                        size    1196647
+github.tarball_from     archive
+
+set port_libfmt         libfmt10
+cmake.module_path-append \
+                        ${prefix}/lib/${port_libfmt}/cmake
+
+depends_build-append    port:bash \
+                        port:bison \
+                        port:re2c
+depends_lib-append      path:lib/pkgconfig/RapidJSON.pc:rapidjson \
+                        port:zlib \
+                        port:${port_libfmt}
+
+set py_ver              3.11
```

Comment:
Does this actually require 3.11? Our default now is 3.12.

Reply:
Until/unless a solution is found to have 3.12 fixed for PPC, I am not using it is any of my ports. (Of course, if you rather not have the port available at all, PR can be closed. But there are multiple existing ports using Python 3.11 and even earlier as the default choice.)

Reply:
3.12 is our default right now. If you don't want to work on it, of course, that's fine. It's a volunteer project.

Reply:
Okay, sure. Anyone else is free to borrow this and submit the port in any form desirable. No objections from my side.